### PR TITLE
Fixed all errors from previous pull!

### PR DESCRIPTION
**Pull request by Tjzabel**
Fixes from PR https://github.com/akath20/crossword-puzzle-solver/pull/2; Made all variables and functions pep 8 compliant. Program works as advertised!

### CODE REVIEW

File: wordSearchSolver.py [hunk 1]
```diff
@@ -49,65 +49,64 @@ def get_input():  # get input two letters
...
+                valid_coordinate_output = valid_coord(add_coordinates(coordinate_to_check, mod_coordinate))
                 # if a valid coordinate
-                if validCoordinateOutput[0]:
+                if valid_coordinate_output[0]:
                     # check if there is a match. If there is one next to first letter, append and break loop
-                    if word_grid[validCoordinateOutput[1]] == user_input[1]:
-                        combinationsDict.setdefault(coordinateToCheck, []).append(
-                            {"coordinate": validCoordinateOutput[1], "modCoordinate": modCoordinate})
-                logging.debug(
-                    "NumPossibilities: {}\nsecond_letter(): {}".format(len(combinationsDict.keys()), combinationsDict)) # combinationsDict.keys() does not work here
```
Comment (Timbone21) [inline wordSearchSolver.py]: finding the len(combination_dict.keys()) does not actually work. Doesn't affect overall file performance, so not sure what to do.

File: wordSearchSolver.py [hunk 2]
```diff
@@ -49,65 +49,64 @@ def get_input():  # get input two letters
...
-                            # if it's the right number, keep going. But if it came to an end, then stop everything and return to user.
-                            return potentialStartCoordinate[0]
-                        currentIterateCount += 1
-            return "Nothing was found."
-        return find_answer(second_letter(first_letter()))
+                    if word_grid[valid_coordinate_output[1]] == user_input[1]:
+                        combinations_dict.setdefault(coordinate_to_check, []).append(
+                            {"coordinate": valid_coordinate_output[1], "mod_coordinate": mod_coordinate})
+        logging.debug(
+            "NumPossibilities: {}\nsecond_letter(): {}".format(len(combinations_dict.keys()), combinations_dict)) # combinations_dict.keys() does not work here
```
Comment (Tjzabel) [inline wordSearchSolver.py:92]: (len(combinations_dict.keys()) does not actually work. Doesn't affect overall file performance, so not sure what to do.

**Comment by akath20** [comment]
How about python 2/3 compatibility? Is this tested?

**Comment by Tjzabel** [comment]
@akath20 completely compatible with at least python 3. When running the file through my command line using python 3, the file completely works. I suppose python 2 may not work since `raw_input` has changed to `input`?

**Comment by akath20** [comment]
test it out with 2/3 using

import sys

if sys.version[0] < 3:
    user_input = raw_input("> ")
else:
    user_input = input("> ")

**Pull request merged**